app/src/pages/playground/playgroundUtils.ts

```diff
@@ -123,6 +128,51 @@ function getOutputFromAttributes(parsedAttributes: unknown) {
   };
 }
 
+/**
+ * Attempts to infer the provider of the model from the model name.
+ * @param modelName the model name to get the provider from
+ * @returns the provider of the model @default AZURE_OPENAI
+ *
+ * NB: Only exported for testing
+ */
+export function getModelProviderFromModelName(
+  modelName: string
+): ModelProvider {
+  if (modelName.includes("gpt") || modelName.includes("o1")) {
+    return "OPENAI";
+  }
```

Comment:
should we just have prefix maps? e.g. OPENAI_MODEL_PREFIXES = ['gpt', 'o1']

Reply:
yup can do that